Providers/Scripts/2.4x-2.5x/Scripts/nxOMSAuditdPlugin.py

```diff
@@ -8,39 +8,58 @@
 import imp
 import codecs
 import subprocess
+import uuid
```

Comment:
I'm not sure that this library is in Python version 2.4 - have you tested this file's use on a system with Python 2.4?